rustls/src/vecbuf.rs

```diff
@@ -142,7 +142,14 @@ impl ChunkVecBuffer {
         Ok(offs)
     }
 
-    fn consume(&mut self, used: usize) {
+    /// Returns the first contiguous chunk of data, or None if empty.
```

Comment:
Nit: since this is a simpler method than both `consume()` and `write_to()`, please move it to the bottom of the impl block.